lib/routes/routeUtapi.js

```diff
@@ -0,0 +1,76 @@
+const url = require('url');
+const http = require('http');
+const { errors } = require('arsenal');
+const { responseJSONBody } = require('arsenal').s3routes.routesUtils;
+const { config } = require('../Config');
+
+function _decodeURI(uri) {
```

Comment:
Since this method is used in routeBackbeat.js, maybe we could go ahead and move to a common util.

Reply:
I agree in general, but I left it in this case since it's just a 3-line function. Looked a little silly in a file all its own x)

Reply:
IIRC this was done because s3cmd replaces spaces with `+`. I don't think we need it for Utapi/Backbeat/Metadata routes.